circles-app/src/lib/stores/wallet.svelte.ts

```
@@ -14,123 +14,116 @@ import { getCirclesConfig } from '$lib/utils/helpers';
-import { environment } from './environment.svelte';
 import { groupMetrics } from './groupMetrics.svelte';
+import { disconnect, getAccount, getConnectors, reconnect } from '@wagmi/core';
+import { config } from '../../config';
+import { settings } from './settings.svelte';
+import type { GroupType } from '@circles-sdk/data';
+import { privateKeyToAccount } from 'viem/accounts';
 
 export const wallet = writable<SdkContractRunner | undefined>();
 
 export const GNOSIS_CHAIN_ID_DEC = 100n;
+export let signer: { address: Address | undefined, privateKey: string | undefined } = $state({ address: undefined, privateKey: undefined });
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'signer' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)